prost-types/src/lib.rs

```diff
@@ -331,6 +331,17 @@ impl fmt::Display for Timestamp {
     }
 }
 
+// Implement stable Eq for Duration.
+// Duration only contains Eq fields.
+// Done manually since protobuf.rs is generated.
+impl Eq for Duration {}
+
+// Implement stable Eq for Any.
+// Any only contains Eq fields.
+// Done manually since protobuf.rs is generated.
+#[cfg(feature = "std")]
+impl Eq for Any {}
```

Comment:
I feel like this may actually be wrong. How do you define equality for something like `Any` since it erases types.

Reply:
Any is just a container for a type descriptor and a raw payload.

It's perfectly reasonable to compare those.

As it stands I need to map the type into my own `Any` to include it in other types that need to be `Eq`